src/pocketmine/level/Level.php

```diff
@@ -1044,6 +1044,15 @@ public function clearCache(bool $force = false){
 		}
 	}
 
+	/**
+	 * @return void
+	 */
+	public function clearBlockCache(int $chunkX, int $chunkZ){
```

Comment:
Native `void` typehint should be used here. The only reason it's doc-commented elsewhere is for backwards compatibility. New API methods should use native types.